PaulKariukiRimiru/Team-timer
# #160582002 User can remove team members

**What does this PR do?**

This PR adds functionality to allow users to remove team members

**Description of Task to be completed?**

As a user of team timer, I should be able to remove team members that are not part of the team anymore

**How should this be manually tested?**

Open the main page and add a new user, then click on the delete icon on that user's name, the name should disappear 

**Any background context you want to provide?**

xxx

**What are the relevant pivotal tracker stories?**

[#160582002 User should be able to remove a team member](https://www.pivotaltracker.com/story/show/160582002)

**Screenshots (if appropriate)**

xxx

**Questions:**

xxx
